perfkitbenchmarker/disk.py

```diff
@@ -172,6 +172,8 @@ class BaseDiskSpec(spec.BaseSpec):
     mount_point: None or string. Directory of mount point.
     num_striped_disks: int. The number of disks to stripe together. If this is
         1, it means no striping will occur. This must be >= 1.
+    os_type: None or string. OS type of the VM to which the disk will be
```

Comment:
@skschneider DeviceId is a Windows-specific concept. It looks like a path, but I don't think it really is a path. I agree with @ehankland that disks don't really have an inherent OS. Maybe it would make more sense to make GetDeviceId be a method of windows_virtual_machine that takes an attached disk as an argument.